client/ios/SnapScreen/Info.plist

```diff
@@ -1,92 +1,98 @@
-    <string>$(EXECUTABLE_NAME)</string>
-    <key>CFBundleIdentifier</key>
-    <string>org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)</string>
-    <key>CFBundleInfoDictionaryVersion</key>
-    <string>6.0</string>
-    <key>CFBundleName</key>
-    <string>$(PRODUCT_NAME)</string>
-    <key>CFBundlePackageType</key>
-    <string>APPL</string>
-    <key>CFBundleShortVersionString</key>
-    <string>1.0</string>
-    <key>CFBundleSignature</key>
-    <string>????</string>
-    <key>CFBundleVersion</key>
-    <string>1</string>
-    <key>LSRequiresIPhoneOS</key>
-    <true/>
-    <key>UILaunchStoryboardName</key>
-    <string>LaunchScreen</string>
-    <key>UIRequiredDeviceCapabilities</key>
-    <array>
-      <string>armv7</string>
-    </array>
-    <key>UISupportedInterfaceOrientations</key>
-    <array>
-      <string>UIInterfaceOrientationPortrait</string>
-      <string>UIInterfaceOrientationLandscapeLeft</string>
-      <string>UIInterfaceOrientationLandscapeRight</string>
-    </array>
-    <key>UIViewControllerBasedStatusBarAppearance</key>
-    <false/>
-    <key>NSLocationWhenInUseUsageDescription</key>
-    <string/>
-    <key>NSAppTransportSecurity</key>
-    <dict>
-      <key>NSExceptionDomains</key>
-      <dict>
-        <key>localhost</key>
-        <dict>
-          <key>NSExceptionAllowsInsecureHTTPLoads</key>
-          <true/>
-        </dict>
-      </dict>
-    </dict>
-    <key>UIAppFonts</key>
-    <array>
-      <string>Andale Mono.ttf</string>
-      <string>Arial Black.ttf</string>
-      <string>Arial.ttf</string>
-      <string>Comic Sans MS.ttf</string>
-      <string>Courier New.ttf</string>
-      <string>Entypo.ttf</string>
-      <string>EvilIcons.ttf</string>
-      <string>FontAwesome.ttf</string>
-      <string>Foundation.ttf</string>
-      <string>Georgia.ttf</string>
-      <string>Ionicons.ttf</string>
-      <string>MaterialIcons.ttf</string>
-      <string>Microsoft Sans Serif.ttf</string>
-      <string>Octicons.ttf</string>
-      <string>Roboto_medium.ttf</string>
-      <string>Roboto.ttf</string>
-      <string>rubicon-icon-font.ttf</string>
-      <string>Rubik-Black.ttf</string>
-      <string>Rubik-BlackItalic.ttf</string>
-      <string>Rubik-Bold.ttf</string>
-      <string>Rubik-BoldItalic.ttf</string>
-      <string>Rubik-Italic.ttf</string>
-      <string>Rubik-Light.ttf</string>
-      <string>Rubik-LightItalic.ttf</string>
-      <string>Rubik-Medium.ttf</string>
-      <string>Rubik-MediumItalic.ttf</string>
-      <string>Rubik-Regular.ttf</string>
-      <string>SanFrancisco.ttf</string>
-      <string>SanFranciscoBold.ttf</string>
-      <string>SanFranciscoThin.ttf</string>
-      <string>SF-UI-Text-Regular.otf</string>
-      <string>Skia.ttf</string>
-      <string>Times New Roman.ttf</string>
-      <string>Zocial.ttf</string>
-    </array>
-  </dict>
-</plist>
\ No newline at end of file
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
```

Comment:
There is a change to allow the app to send http requests to non secure APIs which we need to call our server